.gitignore

```diff
@@ -329,3 +329,9 @@ ASALocalRun/
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
 /logs
+PlanningPoker.Tests/obj/
```

Comment:
are you sure that you need to specify this files explicitly? I can see from your .gitignore file you already have obj and been folder excluded, you can simply remove entries that you have added and remove binary files from git index